skbio/io/format/blast6.py

```diff
@@ -0,0 +1,287 @@
+separated by the tab character.
+
+An example BLAST+6-formatted file comparing two protein sequences, taken
+from [1]_::
+
+    moaC    gi|15800534|ref|NP_286546.1|    100.00  161 0   0   1   161 1
+    161 3e-114  330
+    moaC    gi|170768970|ref|ZP_02903423.1| 99.38   161 1   0   1   161 1
+    161 9e-114  329
+
+Format Support
+--------------
+**Has Sniffer: No**
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |No    |:mod:`pandas.DataFrame`                                        |
++------+------+---------------------------------------------------------------+
+
+Format Specification
+--------------------
+BLAST+6 format is a tabular text-based format produced by BLAST+ format 6,
+containing data related to passed sequences, tab-separated, with no column
+labels or headers of any kind.
+
+When blasting sequences using Ubuntu's command line, the user may specify what
+data is returned.
+
+For example::
+
+    $ blastp -query queries.fasta -subject subjects.fasta -evalue 0.0001\
+    -max_target_seqs 2 -outfmt "6 qseqid sseqid bitscore qstart sstart"
+    $ query1	subject2	16.9	1	3
+
+In this case, the blastp command returns custom values, which were specified
+by the user to be qseqid, sseqid, bitscore, qstart, and sstart. These names
+are retained, but as the returned values show, these names do not mark the
+columns. The user can also pass ``-outfmt 6`` instead, which will set the
+columns to their default values.
+
+For example::
+
+    $ blastp -query queries.fasta -subject subjects.fasta -evalue 0.0001\
+    -max_target_seqs 2 -outfmt 6
+    $ query1	subject2	100.00	8	0	0	1	8	3	10	9e-05	16.9
+
+If the user does not pass custom values, the blastp command will use the
+default columns, which are qseqid, sseqid, pident, length, mismatch, gapopen,
+qstart, qend, sstart, send, evalue, and bitscore.
+
+Types of Data
+^^^^^^^^^^^^^
++-----------+------------------------------------+-----+
+|Name       |Description                         |Type |
++===========+====================================+=====+
+|qseqid     |Query Seq-id                        |str  |
++-----------+------------------------------------+-----+
+|qgi        |Query GI                            |int  |
++-----------+------------------------------------+-----+
+|qacc       |Query accesion                      |str  |
++-----------+------------------------------------+-----+
+|qaccver    |Query accesion.version              |str  |
++-----------+------------------------------------+-----+
+|qlen       |Query sequence length               |int  |
++-----------+------------------------------------+-----+
+|sseqid     |Subject Seq-id                      |str  |
++-----------+------------------------------------+-----+
+|sallseqid  |All subject Seq-id(s), separated by\|str  |
```

Comment:
The second column doesn't render correctly ("bya" instead of "by a"). I think you can remove the line continuation here. There are a few more cells like this below that'll need to be modified as well.